packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/summary.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { shape, string, func } from 'prop-types';
+import { Edit2 as EditIcon } from 'react-feather';
+import { useSummary } from '@magento/peregrine/lib/talons/CheckoutPage/PaymentInformation/useSummary';
+
+import Icon from '../../Icon';
+import { mergeClasses } from '../../../classify';
+
+import summaryGQLOperations from './summary.gql';
+
+import defaultClasses from './summary.css';
+
+const Summary = props => {
+    const { classes: propClasses, onEdit, paymentNonce } = props;
+
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    const talonProps = useSummary({ operations: summaryGQLOperations });
+
+    const { billingAddress, isBillingAddressSame } = talonProps;
+
+    const billingAddressSummary =
+        !isBillingAddressSame && billingAddress ? (
+            /**
+             * Added ID for tests
+             */
+            <div
+                id={'addressSummary'}
+                className={classes.address_summary_container}
+            >
+                <div>
+                    <span className={classes.first_name}>
+                        {billingAddress.firstName}
+                    </span>
+                    <span className={classes.last_name}>
+                        {billingAddress.lastName}
+                    </span>
+                </div>
+                <div>
+                    <span className={classes.street1}>
+                        {billingAddress.street1}
+                    </span>
+                    <span className={classes.street2}>
+                        {billingAddress.street2}
+                    </span>
+                    <span className={classes.city}>{billingAddress.city}</span>
+                    <span className={classes.state}>
+                        {billingAddress.state}
+                    </span>
+                </div>
+                <div>
+                    <span className={classes.postalCode}>
+                        {billingAddress.postalCode}
+                    </span>
+                    <span className={classes.country}>
+                        {billingAddress.country}
+                    </span>
+                </div>
+            </div>
+        ) : null;
+
+    return (
+        <div className={classes.root}>
+            <div className={classes.heading_container}>
+                <h5 className={classes.heading}>Payment Information</h5>
+                <button className={classes.edit_button} onClick={onEdit}>
+                    <Icon size={16} src={EditIcon} attrs={{ fill: 'black' }} />
```

Comment:
Do you need fill here? I think we talked about this in the developer review but I don't remember what we said. If you do need it, this doesn't seem to be applying the fill anyways.

Reply:
This is the old syntax. I made a note about changing this to the new syntax. I don't think we need the fill anymore.